workers/unity/Packages/io.improbable.gdk.tools/GenerateCode.cs

```diff
@@ -70,6 +71,7 @@ private static void GenerateMenu()
 
         private static void Generate()
         {
+            var errorMessage = new StringBuilder();
```

Comment:
Prefer that this is declared closer to where it is used (currently ~50 lines apart)

Reply:
Fair enough. Wanted to keep it on top of the `Generate` method but having it closer makes sense.